docs/cloud-native-security/session-view.asciidoc

```diff
@@ -15,7 +15,7 @@ and investigating session activity on your Linux infrastructure and understandin
 
 [float]
 [[session-view-data]]
-====== Session View displays:
+== Session View displays
```

Comment:
This is sort of an odd heading because it's also acting as a lead-in sentence to the list below, as in "Session View displays the following things." Because it's unexpected for a heading to also act like a sentence, I was reading it more like "This is a section about the Session View displays" but I don't think that's a thing. Anyway, maybe something like:
```suggestion
== Session View features
```

And/or give the list a proper lead-in sentence, but that feels like adding a sentence just to add a sentence so maybe just focus on the heading.

Reply:
I would prefer to leave this as `Session View` to align with the other titles of investigation tools -- `Timeline`, `Visual event analyzer`, etc. It would seem a little misaligned if we did not, IMO.

Reply:
@jmikell821 to clarify, are you suggesting that [this section](https://security-docs_bk_5736.docs-preview.app.elstc.co/guide/en/security/master/session-view.html#session-view-data) be renamed from "Session view features" to "Session View"?

Reply:
@nastasha-solomon oops — so sorry, my eyes crossed here. I think we can remove this subheading and have a lead-in sentence instead.

Reply:
All good and thanks, @jmikell821! I've made your requested changes in [284f14b](https://github.com/elastic/security-docs/pull/5736/commits/284f14b651bb4d093f30ff0beaaa1a96a7436d5f).